src/FapiApi.php

```diff
@@ -272,4 +224,42 @@ public function isVoucherSecurityValid( $voucher, $itemTemplate, $time, $expecte
 		return SecurityValidator::isVoucherSecurityValid( $voucher, $itemTemplate, $time, $expectedSecurity );
 	}
 
+    /**
+     * @param string $method
+     * @param string $remoteUrl
+     * @param string|null $bodyData
+     * @param int $retries
+     * @return mixed
+     */
+    private function retryRequest($method, $remoteUrl, $bodyData = null, $retries = 3) {
+        $requestData = array(
+           'method'  => $method == 'GET' ? 'GET' : 'POST',
+           'headers' => $this->createHeaders(),
+           'timeout' => 30,
+           'connection_timeout' => 30,
+        );
+
+        if ($method == 'POST') {
```

Comment:
Tohle vyhoď je to hnusný hack... 
nech tam jen ty gety a klidně tam dej validaci že jaká koliv jiná metoda do retry vyhodí exception, aby to nikdo neopužíval a post/put/delete requesty. 
Ta logika je hrozný side effekt...

Reply:
je to udělané jenom na get requesty